FluentUI/build.gradle

```diff
@@ -1,112 +1,90 @@
-/*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License.
```

Comment:
nit: Need this copyright statement

Reply:
Corrected. This has happened due to re-naming of existing files and adding new files for some modules